first_hour_on_terra/working_with_data_collections.ipynb

```diff
@@ -0,0 +1,476 @@
+    " style=input_style\n",
+    ")\n",
+    "output_workspace_id = widgets.Text()\n",
+    "input_description = widgets.Text(\n",
+    " placeholder=\"<DESCRIPTION>\",\n",
+    " description=\"Description:\",\n",
+    " style=input_style\n",
+    " )\n",
+    "input_name = widgets.Text(\n",
+    " placeholder=\"<RESOURCE_NAME>\",\n",
+    " description=\"Resource Name:\",\n",
+    " style=input_style\n",
+    " )\n",
+    "display(input_workspace_id)\n",
+    "display(input_description)\n",
+    "display(input_name)\n",
+    "\n",
+    "def bind_input_to_output(e):\n",
+    "    output_workspace_id.value = input_workspace_id.value\n",
+    "\n",
+    "# define a function for the button to call\n",
+    "def button_click_event(b):\n",
+    "    with output:\n",
+    "        global buttonOutput\n",
+    "        terraCommand = f\"terra workspace create --id={input_workspace_id.value} --description=\\\"{input_description.value}\\\" --name={input_name.value}\"\"\".format(input_workspace_id.value,input_description.value,input_name.value)       \n",
+    "        print(\"Running command to create workspace...\")\n",
+    "        print(terraCommand)\n",
+    "        result = subprocess.run([\"terra\",\"workspace\",\"create\",f\"--id={input_workspace_id.value}\",f\"--description=\\\"{input_description.value}\\\"\",f\"--name={input_name.value}\"],\n",
+    "                                capture_output=True,text=True)\n",
+    "        print(\"Your workspace will be ready momentarily...\")\n",
+    "        print(result.stdout)\n",
+    "\n",
+    "# get a reference to the widget output\n",
+    "output = widgets.Output()\n",
+    "\n",
+    "input_workspace_id.observe(bind_input_to_output)\n",
+    "\n",
+    "button = widgets.Button(\n",
+    "    description='Create workspace',\n",
+    "    disabled=False,\n",
+    "    button_style='',\n",
+    "    tooltip='Click to create a new workspace',\n",
+    "    icon='check',\n",
+    "    layout=widgets.Layout(width='50%', height='40px')\n",
+    ")\n",
+    "\n",
+    "#bind the button_click_event to the button call event\n",
+    "button.on_click(button_click_event)\n",
+    "\n",
+    "# show the current state of the output\n",
+    "print(buttonOutput)\n",
+    "\n",
+    "#display the button\n",
+    "display(button, output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
+    "tags": []
+   },
+   "source": [
+    "### Add resources to data collection\n",
+    "\n",
+    "Add new controlled and/or referenced resources to the workspace created in the previous step using the Terra CLI or the Verily Workbench Resources tab. To easily add resources, use the widgets provided in [../workspace_resource_examples.ipynb](../workspace_resource_examples.ipynb../workspace_resource_examples.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Convert workspace to data collection\n",
```

Comment:
Same comment as above about making it clear what the user is expected to input. Please add instructions here that tells the user that they're expected to input the SAME workspace id, a description (is this expected to be same or different?), and what version is expected to be. And again please give examples of good names.